### PR TITLE
Changes to make the cluster sub package golint-able

### PR DESCRIPTION
**Pull request by lenko-d**
Issue: Enable golint on the code base #4098

### CODE REVIEW

File: cluster/rpc.go [hunk 1]
```diff
@@ -16,12 +16,22 @@ type MapShardRequest struct {
 	pb internal.MapShardRequest
 }
 
-func (m *MapShardRequest) ShardID() uint64  { return m.pb.GetShardID() }
-func (m *MapShardRequest) Query() string    { return m.pb.GetQuery() }
+// ShardID of the storage
```
Comment (otoolep) [inline cluster/rpc.go]: Isn't this the shard ID of the map request?

File: cluster/rpc.go [hunk 2]
```diff
@@ -16,12 +16,22 @@ type MapShardRequest struct {
...
+// Query returns the protocol buffers' query string
+func (m *MapShardRequest) Query() string { return m.pb.GetQuery() }
+
+// ChunkSize - proto buff's chunk size
 func (m *MapShardRequest) ChunkSize() int32 { return m.pb.GetChunkSize() }
 
-func (m *MapShardRequest) SetShardID(id uint64)         { m.pb.ShardID = &id }
-func (m *MapShardRequest) SetQuery(query string)        { m.pb.Query = &query }
+// SetShardID sets the shard id
+func (m *MapShardRequest) SetShardID(id uint64) { m.pb.ShardID = &id }
```
Comment (otoolep) [inline cluster/rpc.go]: Ditto. I think the comments should reference the "Shard map request". A "shard map request" in our system is a request to access a shard on another node.

**Comment by otoolep** [comment]
Thanks @lenko-d -- have you signed the CLA?

**Comment by lenko-d** [comment]
I fixed the comments. Please review. CLA signed.

**Comment by otoolep** [comment]
Looks good @lenko-d -- thanks.

Comments only, confirmed CLA signed, merging now.

**Pull request merged**